### DISCUSSION TITLE
Townsend model string orientation adjustment

### DISCUSSION THREAD

**Discussion by lochermichael** [pvlib/pvlib-python#1625] [General]
I see that the Townsend snow loss model was added in a recent update. However, I believe there has been an unpublished addition to the model since the whitepapers were released back in 2010/2011. Specifically, the addition of a "string orientation adjustment" based on whether a single string or multiple strings are present in the array upslope direction. This is from communication with the author, who presented a sample calculation workbook. A screenshot is attached. The proposed multipliers are 1.0 for a single string and 0.75 for multiple parallel strings.

[Previous screenshot removed]

**Comment by cwhanse** [comment]
@lochermichael this modifier simply multiplies the monthly loss fraction, is that right? If so, this would be a simple addition to the code. Do you think we could obtain any documentation of this extension of the published model?

**Comment by lochermichael** [comment]
Correct, it is just a simple multiplier. The challenge isn't necessarily in implementing it, but in obtaining the documentation to support adding it. I did try using my previous contact info for T. Townsend, but it seems it is no longer active and I'm not sure I have an alternative, unfortunately.

**Comment by cwhanse** [comment]
The string_factor is added with #1636

**Comment by wedize** [comment]
I was wondering if the calculation workbook  of the Townsend snow losses model is available to public use. If so, could anyone share the download link or details for accessing it. Your assistance would be greatly appreciated. Thank you.
Best regards,
wedize

**Reply by cwhanse** [reply]
To my knowledge the spreadsheet is not public. You would need to get it from Mr. Townsend.

**Comment by wedize** [comment]
I am writing to seek clarification regarding specific variables within the Townsend Snow Losses Model.
In the model , it clear that the variables "**relative_humidity**" and "**temp_air**" are specified as monthly averages. However, I am interested in undestanding the treatment of the two other variables the "**total_snow**" and "**global_poa**" .
Could anyone provide me information on whether "**total_snow**" and "**global_poa**"are considered as monthly averages or the sum of the respective values for each month?
Furthermore, I would like to inquire about the model's validity when applied to data collected on an **hourly basis**. Does the Townsend Snow Losses Model maintain its accuracy and reliability when working with high-frequency data, or is it primarily designed for monthly data inputs?
Thank you .
Best regards,
wedize

**Reply by kandersolar** [reply]
Snow and insolation are understood to be monthly totals (not averages) as far as I know.  It's not clear to me that average snowfall is a meaningful concept anyway; wouldn't it depend on the time scale of the snowfall data?  

The model was designed specifically for monthly inputs.  I have not seen any assessment of its performance at other time scales and would not expect it to behave reasonably at hourly scale.  For higher resolution loss estimates, you might consider using the NREL snow loss model, which is used often with hourly data: https://pvlib-python.readthedocs.io/en/stable/reference/effects_on_pv_system_output/snow.html

**Reply by wedize** [reply]
@kandersolar Thankk you for the response and for the clarifications. I will consider the NREL snow loss model for hourly data.
